templates/secrets_st2chatops.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-{{- range $env, $value := .Values.st2chatops.env }}
+{{- range $env, $value := .Values.secrets.st2.st2chatops.env }}
```

Comment:
I think there is at least an agreement about the :100: need of raw `ENV` for every Pod and that's super helpful.
Can we keep raw ENV functionality in this PR to be able to merge it asap, but split the `st2chatops` secrets ENV enhancement into another PR?